pynvml_utils/tests/test_nvml.py

```diff
@@ -92,7 +92,7 @@ def pci_info(ngpus, handles):
 # Test pynvml.nvmlSystemGetNVMLVersion
 def test_nvmlSystemGetNVMLVersion(nvml):
```

Comment:
Do they ship tests too?

Reply:
Not that I know of. I only get a single file when I install nvidia-ml-py. E.g.:

```
conda/envs/.../lib/python3.12/site-packages/pynvml.py
```

Reply:
It could be their `sdist` has them, but their packages do not

Reply:
Looks like they don't